docs/vc-http-api.yml

```diff
@@ -592,6 +651,91 @@ components:
           $ref: "#/components/schemas/PresentCredentialOptions"
     ProvePresentationResponse:
       $ref: "#/components/schemas/VerifiablePresentation"
+    NotifyPresentationAvailableRequest:
+      type: object
+      properties:
+        query:
+          type: object
+          description: See https://w3c-ccg.github.io/vp-request-spec/#format
+          properties:
```

Comment:
Shouldn't the caller DID be in this message? (or maybe it is somewhere else in the request butt I missed it).
Seems like a good opportunity to DID-Auth to this holder (server) - is that required? desired? anti-pattern?

Reply:
> Shouldn't the caller DID be in this message? 
Oops, I meant presenter's id URI.
If it is included, is there some sort of authentication required associated with this id uri that the spec needs to define?

Reply:
https://w3c-ccg.github.io/vp-request-spec/#example-1-a-sample-query

```
recipients: [
    // an optional key agreement key for encrypting the response if
    // this is supported
  ]
```

Reply:
I'm trying real hard not to invent new data models here, so only using things in the vc data model or vp request spec.